beater/sipcmbeat.go

```diff
@@ -163,6 +165,9 @@ waitsig:
 			for bt.evIdx != last {
 				ev, nxtIdx, err := bt.evRing.Get(bt.evIdx)
 				if ev != nil {
+					if err := ev.EncryptIP(bt.anonymKey); err != nil {
+						fmt.Fprintf(os.Stderr, "could not anonymize IP address; error: %s\n", err)
+					}
```

Comment:
Obsolete now? (since it's done per field)

BTW: another reason for having it when generating the ES event fields is to be more consistent with the URI encryption. The encrypted URIs will most likely be bigger then the original ones and there will not be space for them in the event (all "strings" are stored as pointer in the event buf, where everything is "compacted" together).

https://github.com/intuitivelabs/sipcmbeat/pull/1#commitcomment-45383865